src/containers/Review/assignment/AssignmentSectionRow.tsx

```diff
@@ -128,7 +126,7 @@ const AssignmentSectionRow: React.FC<AssignmentSectionRowProps> = ({
           </Label>
         </Grid.Column>
         <Grid.Column className="centered-flex-box-row" width={9}>
-          {originalAssignee ? (
+          {originalAssignee && assignmentOptions.isSubmitted ? (
```

Comment:
This was a **new** problem, happens after the selection of the assigned reviewer, it was interpreting the same way as an existing Assignment on the database, so I added this flag to determine when this is submitted assignment or un-submitted